core/src/main/scala/org/apache/spark/TestUtils.scala

```diff
@@ -491,6 +491,27 @@ private[spark] object TestUtils {
       EnumSet.of(OWNER_READ, OWNER_EXECUTE, OWNER_WRITE))
     file.getPath
   }
+
+  def withConf[T](confPairs: (String, String)*)(f: => T): T = {
```

Comment:
It is nice to add some comments for the usage of this function, we see that some other public functions had before in this file. Better to follow the same.

Reply:
Sure. Will do.